pkgs/development/python-modules/pki-core/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, buildPythonPackage, sphinx, rpmextract, requests,
+  six }:
+
+buildPythonPackage rec {
+  name = "pki-core-${version}";
+  version = "10.10.5-6";
+
+  rpmFile = "${name}-7.fc25.src.rpm";
+
+  src = fetchurl {
+    url = "mirror://fedora/linux/releases/25/Everything/source/tree/Packages/p/${rpmFile}";
```

Comment:
```suggestion
  src = fetchurl {
    url = "mirror://fedora/linux/releases/25/Everything/source/tree/Packages/p/pki-core-${version}-7.fc25.src.rpm";
```

Reply:
Removed pki-core entirely, as it is not in use.